docs/concepts/flows.md

```diff
@@ -1169,3 +1169,21 @@ prefect flow-run cancel 'a55a4804-9e3c-4042-8b59-b3b6b7618736'
 From the UI you can cancel a flow run by navigating to the flow run's detail page and clicking the `Cancel` button in the upper right corner.
 
 ![Prefect UI](/img/ui/flow-run-cancellation-ui.png)
+
+## Timeouts
+
+Flow timeouts are used to prevent unintentional long-running flows. When the duration of execution for a flow exceeds the duration specified in the timeout, a timeout exception will be raised and the flow will be marked as failed. In the UI, the flow will be visibly designated as `TimedOut`.
+
+Timeout durations are specified using the `timeout_seconds` keyword argument.
+
+```python
+from prefect import flow, get_run_logger
+import time
+
+@flow(timeout_seconds=1)
+def show_timeouts():
+    logger = get_run_logger()
+    logger.info("I will execute")
+    time.sleep(5)
+    logger.info("I will not execute")
```

Comment:
```suggestion
    logger.info("I will not execute")
    
if __name__ == '__main__':
    show_timeouts()
```

I tested this out and the example works perfectly.  My only question is whether we'd want our examples to include the `if __name__ == '__main__':` block or if that's just too much extra noise.

Reply:
The only reason I noticed is that I just pasted this in a blank file, so I thought others might try the same thing

Reply:
There are a bunch of places in this doc that don't have the block with the function call and the tasks page is the same, so I tried to add some consistency - but didn't add the function call here.

Reply:
Asked for a re-review given the additional changes.